src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/TicketSearchService.cs

```diff
@@ -38,6 +42,9 @@ public TicketSearchService(IBus bus, ITicketInvoiceSearchService invoiceSearchSe
             IEnumerable<Invoice>? response = await _invoiceSearchService.SearchAsync(ticketNumber, issuedTime, cancellationToken);
             var invoices = response.ToList();
 
+            // TCVP-2651 Filter results to only MVA/MVAR violation tickets
+            invoices = invoices.Where(_ => _.Act == _mva || _.Act == _mvar).ToList();
+
```

Comment:
It might be better to avoid creating an extra list to filter out items like:

var invoices = response.Where(invoice => invoice.Act == _mva || invoice.Act == _mvar).ToList();